individual.h

```diff
@@ -42,6 +43,9 @@ class individual
   ///Sets the fitness of an ind
   void set_fitness(double fitness) {m_fitness = fitness;}
 
+  ///Set the input values of an individual
+  void set_input(std::vector<double> input) {m_input_values = input;}
```

Comment:
We want it, but set is to much of a general term, and indeed in c++ it is a "common-type" function. It is better to name it something else that is more specific, in my opinion, because then we call it only when we want to execute a specific operation, such as assigning inputs to an individual, and not whenever we want to set the input to something (avoid the temptation of using this function everrywhere). I might be wrong on this point, but I feel this is the way I would proceed.
Remeber if we give all memebers of a `class` a getter and a setter we basically make a `class` a `structure`, whihc defeats the purpose a bit.